src/Umbraco.Web/PropertyEditors/MediaPicker3Configuration.cs

```diff
@@ -57,4 +61,40 @@ public class CropConfiguration
             public int Height { get; set; }
         }
     }
+
+    internal static class MediaPicker3ConfigurationExtensions
+    {
+        /// <summary>
+        /// Applies the configuration to ensure only valid crops are kept and have the correct width/height.
+        /// </summary>
+        /// <param name="configuration">The configuration.</param>
+        public static void ApplyConfiguration(this ImageCropperValue imageCropperValue, MediaPicker3Configuration configuration)
+        {
+            var crops = new List<ImageCropperCrop>();
+
+            var configuredCrops = configuration?.Crops;
+            if (configuredCrops != null)
+            {
+                foreach (var configuredCrop in configuredCrops)
+                {
+                    var crop = imageCropperValue.Crops?.FirstOrDefault(x => x.Alias == configuredCrop.Alias);
+
+                    crops.Add(new ImageCropperCrop
+                    {
+                        Alias = configuredCrop.Alias,
+                        Width = configuredCrop.Width,
+                        Height = configuredCrop.Height,
+                        Coordinates = crop?.Coordinates
+                    });
+                }
+            }
+
+            imageCropperValue.Crops = crops;
+
+            if (configuration?.EnableLocalFocalPoint == false)
```

Comment:
We need to check for this configuration setting, as the picker always stores the default focal point (0.5, 0.5). Removing this together with the crop width/height would also reduce the size of data stored!